### PR TITLE
UCT/IB: complete RX CQ cleanup on device failure

### PR DESCRIPTION
**Pull request by evgeny-leksikov**
fix a stuck on CQ resources cleanup in case of device fatal error

### CODE REVIEW

**Review by dmitrygx — APPROVED** [review]

File: src/uct/ib/base/ib_device.c [hunk 1]
```diff
@@ -298,6 +298,10 @@ static void uct_ib_async_event_handler(int fd, void *arg)
         level = UCS_LOG_LEVEL_DEBUG;
         break;
     case IBV_EVENT_DEVICE_FATAL:
+        ucs_assert(event.element.port_num == 0);
```
Comment (yosefe) [inline src/uct/ib/base/ib_device.c]: why needed?
maybe just pass 0 instead of event.element.port_num?

**Review by yosefe — COMMENTED** [review]
pls add a similar w/a to integration4 as well

File: src/uct/ib/base/ib_device.c (hunk 1 above)
Comment (evgeny-leksikov) [reply src/uct/ib/base/ib_device.c]: done

**Comment by evgeny-leksikov** [comment]
> pls add a similar w/a to integration4 as well

ok

**Review by yosefe — APPROVED** [review]

**Pull request merged**